pychromecast/discovery.py

```diff
@@ -190,13 +202,14 @@ def discover_listed_chromecasts(
       A list of chromecast services matching the criteria,
       or an empty list if no matching chromecasts were found.
       A service browser to keep the Chromecast mDNS data updated. When updates
-      are (no longer) needed, pass the broswer object to
-      pychromecast.discovery.stop_discover().
+      are (no longer) needed, pass the browser object to
+      pychromecast.discovery.stop_discovery().
```

Comment:
Will stop discovery stop the zeroconf instance that the browser is connected to ? It should do so only if the browser is not reusing a zeroconf instance?

Reply:
HomeAssistant's shared zeroconf instance overrides the `close()` method so it's not a problem.